addons/product/views/product_views.xml

```diff
@@ -614,11 +614,11 @@ action = {
                 <searchpanel>
                     <field name="categ_id"
                            string="Product Category"
-                           icon="fa-th-list"
-                           enable_counters="1"/>
+                           icon="fa-th-list"/>
                     <field name="product_template_attribute_value_ids"
                            string="Attributes"
                            icon="fa-th-list"
+                           domain="[('ptav_active', '=', True), ('product_tmpl_id.active', '=', True)]"
                            enable_counters="1"
```

Comment:
Isn't it on the ptav filter that you want to remove the counters ?

Reply:
We only need to remove counter from `category` type section not `filter` type section [see](https://github.com/odoo/odoo/blob/17.0/addons/web/static/src/search/search_arch_parser.js#L342)
Having counter on filter type does not cause any issue I guess but we can remove it if it is not very useful :thinking: 
[Difference b/t category and filter type](https://github.com/odoo/odoo/blob/17.0/addons/web/static/src/search/search_arch_parser.js#L301)